tests/fixture/bootstrapmonitor/README.md

```diff
@@ -48,9 +56,16 @@ and initiates a new test when one is found.
    - The network targeted by the test is determined by the value of
      the `AVAGO_NETWORK_NAME` env var set for the avalanchego
      container.
-   - Whether state sync is enabled is determined by the value of the
-     `AVAGO_CHAIN_CONFIG_CONTENT` env var set for the avalanchego
-     container.
+   - By default, the sync mode will be `full-sync`. The other sync
```

Comment:
Above, it says that C-Chain State Sync is the default mode, which seems to be contradicted here.